tests/wallet/rpc/test_dl_wallet_rpc.py

```diff
@@ -144,9 +154,11 @@ async def is_singleton_confirmed(rpc_client, lid) -> bool:
 
             await client_2.dl_track_new(launcher_id)
 
-            async def is_singleton_generation(rpc_client, lid, generation) -> bool:
+            async def is_singleton_generation(rpc_client: WalletRpcClient, lid: bytes32, generation: int) -> bool:
                 if await is_singleton_confirmed(rpc_client, lid):
                     rec = await rpc_client.dl_latest_singleton(lid)
+                    if rec is None:
+                        raise Exception("No latest singleton for: {lid!r}")
```

Comment:
The next line would have raised an `AttributeError` so at the level of 'an exception is raised' there is no change here.